### DISCUSSION TITLE
Can't launch Surge XT<title>

### DISCUSSION THREAD

**Discussion by oneandrew9** [community/community#123689] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I installed SurgeXT using Software Manager. It seems to install with progress bars and ends on the Launch screen. When I click Launch, it does not start. Nothing happens, no error messages. I observe it created sub-directories but they are empty (no files). I have done this several times and I do not know what else to do. Can you help?

**Comment by devxMani** [comment]
Try launching from the terminal,  this might provide more information on what's going wrong type the command surgeXT followed by the path if it's installed in a non-standard location.

**Comment by oneandrew9** [comment, marked as answer]
Hi
Thanks for getting back to me. I downloaded the Debian version form GitHub and that one worked perfectly. 
There is something wrong with the way Software Manager was doing it, at least on my computer. 
Have a nice day,
best,andrew
    On Monday, May 13, 2024 at 06:39:55 PM EDT, MANI ***@***.***> wrote:  
 
 


Try launching from the terminal, this might provide more information on what's going wrong type the command surgeXT followed by the path if it's installed in a non-standard location.
